### PR TITLE
DOC: tech edit for software-update-server.rst

### PR DESCRIPTION
**Pull request by pohly**
This is PR #146 rebased onto current master. I'm creating it because
David had some problems while wrangling with git.

@dbkinder please do a quick check whether I rebased correctlty and then we can merge it.

### CODE REVIEW

**Comment by dbkinder** [comment]
+1  other than forgetting to add [skip ci] to the PR message :)
Thanks for your help!  I did figure out my git issue...

File: doc/howtos/software-update-server.rst [hunk 1]
```diff
@@ -158,31 +157,45 @@ customized image, by:
...
+to the set of images that are buildable. This new image
+contains the
+``os-core`` bundle,  the ``sudo_bundle`` bundle from Step 1, and whatever
 else might have been added.
 
-The main difference, compared to the approach taken in Step 0 is that
-this approach allows for removing the content of the bundle without having to
+If you have ``sudo`` as part of ``os-core``, then you:
+- remove it from the bundle and publish the update, *all* the devices will have sudo removed.
+- add it tp the bundle and publish the update, *all* the devices will have sudo installed.
```
Comment (barbieri) [inline doc/howtos/software-update-server.rst]: typo "tp" instead of "to"

**Comment by pohly** [comment]
Darn! I need to submit more doc PRs, otherwise I'll keep forgetting the [skip ci] :-}

File: doc/howtos/software-update-server.rst (hunk 1 above)
Comment (pohly) [inline doc/howtos/software-update-server.rst]: Fixed before merging.

**Pull request merged**